src/Frontend/Components/Note/Note.js

```diff
@@ -0,0 +1,78 @@
+import React, { useState } from "react";
+import moment from "moment";
+
+import { Modal } from "../Modal/Modal";
+import { useNote } from "../../contexts/NoteContext";
+import "./note.css";
+import { useAuth } from "../../contexts/AuthContext";
+
```

Comment:
You can try creating an index.js in Contexts folder and then import - useNote and useAuth fron that index.js file.This would decrease the lines of code and increase readability.